Build/docker/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# (C) Copyright IBM Corporation 2017.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This Dockerfile in Build/docker dir is used to create an image with
+# AdoptOpenJDK jdk binary installed. Basic test dependent executions
+# are installed during the building process.
+#
+# Build example: `docker build -t adoptopenjdk-test .`
+#
+# This Dockerfile builds image based on adoptopenjdk/openjdk8:latest.
+# If you want to build image based on other images, please use
+# `--build-arg list` to specify your base image
+#
+# Build example: `docker build --build-arg IMAGE_NAME=<image_name> --build-arg IMAGE_VERSION=<image_version >-t adoptopenjdk-test .`
+
+ARG IMAGE_NAME=adoptopenjdk/openjdk8
+ARG IMAGE_VERSION=latest
+
+FROM ${IMAGE_NAME}:${IMAGE_VERSION}
+
+MAINTAINER Tianyu Zuo <tianyu@ca.ibm.com>
```

Comment:
You will likely not be the only maintainer of this file, so may choose to leave this out (as we have done in the Dockerfiles at the OpenJ9 project).

Reply:
yes, PR updated